x-pack/plugins/security_solution/public/timelines/components/open_timeline/index.tsx

```diff
@@ -197,6 +200,10 @@ export const StatefulOpenTimelineComponent = React.memo<OpenTimelineOwnProps>(
 
     const deleteTimelines: DeleteTimelines = useCallback(
       async (timelineIds: string[]) => {
+        startTransaction({
+          name: timelineIds.length > 1 ? TIMELINE_ACTIONS.DELETE : TIMELINE_ACTIONS.BULK_DELETE,
```

Comment:
I think these should be swapped. `timelineIds.length > 1 ? TIMELINE_ACTIONS.BULK_DELETE : TIMELINE_ACTIONS.DELETE`

Reply:
good catch!